src/cdk-experimental/menu/menu-bar.ts

```diff
@@ -28,10 +44,143 @@ import {CDK_MENU, Menu} from './menu-interface';
     {provide: CDK_MENU, useExisting: CdkMenuBar},
   ],
 })
-export class CdkMenuBar extends CdkMenuGroup implements Menu {
+export class CdkMenuBar extends CdkMenuGroup implements Menu, AfterContentInit, OnDestroy {
   /**
    * Sets the aria-orientation attribute and determines where menus will be opened.
    * Does not affect styling/layout.
    */
   @Input('cdkMenuBarOrientation') orientation: 'horizontal' | 'vertical' = 'horizontal';
+
+  /** All child MenuItem elements nested in this MenuBar. */
+  @ContentChildren(CdkMenuItem, {descendants: true})
+  private readonly _allItems: QueryList<CdkMenuItem>;
+
+  /** Event emitter used to end the subscription to document click events. */
+  private readonly _destroyDocumentSubscription: EventEmitter<void> = new EventEmitter();
```

Comment:
When it's not an `@Output`, you can just use `Subject` from rxjs instead of `EventEmitter`

Reply:
Done. Is there a particular reason?